modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -94,3 +94,24 @@ Using `salt-thin` can be enabled as a fallback method, but it requires Python 3
 This method is not recommended nor supported and exists for development purposes only.
 Set [option]``web.ssh_use_salt_thin`` to `true` in the [path]``/etc/rhn/rhn.conf`` configuration file.
 ====
+
+== Extending Salt Bundle functionality with Python packages using pip
```

Comment:
I'm sorry that I must start with this again.
```suggestion
== Extend Salt Bundle with Python packages using pip
```
In titles we prefer verbal form.  I think this is short for "How to extend the Salt Bundle ...".  The -ing form is reserved for procedure titles.  That's how Lana argued.  I'd like to stay with this style.  If you agree here, we also should change the .changelog entry.